src/internal/connector/onedrive/item.go

```diff
@@ -191,7 +191,7 @@ func oneDriveItemMetaInfo(
 
 	perm, err := service.Client().DrivesById(driveID).ItemsById(*itemID).Permissions().Get(ctx, nil)
 	if err != nil {
-		return Metadata{}, err
+		return Metadata{}, support.ConnectorStackErrorTraceWrap(err, "fetching item permissions")
```

Comment:
We should probably use `clues` here instead.

Reply:
excellent point, thanks for the reminder